src/Glpi/Asset/AssetDefinition.php

```diff
@@ -478,12 +478,27 @@ private function prepareInput(array $input): array|bool
 
     public function post_addItem()
     {
+        /** @var \DBmysql $DB */
+        global $DB;
+
         if ($this->isActive()) {
             $this->syncProfilesRights();
 
             // Force menu refresh when active state change
             unset($_SESSION['menu']);
         }
+
+        // Add default display preferences for the new asset definition
+        // Status, Manufacturer, Serial, Type, Model, Location, Last Update
+        $prefs = [31, 23, 5, 4, 40, 3, 19];
```

Comment:
I just reorder preferences to match the default computer preferences.

```suggestion
        $prefs = [
            4, // Name
            40, // Model
            5, // Serial
            23, // Manufacturer
            31, // Status
            3, // Location
            19, // Last Update
        ];
```